<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorvusVerification</class>
 <widget class="QWidget" name="CorvusVerification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4">
    <widget class="QPushButton" name="pB_LoadConfigure">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QPushButton" name="pB_Reset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pB_Configure">
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="LE_FilePath">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QPushButton" name="pB_Start">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QPushButton" name="pB_Continue">
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QPushButton" name="pB_Stop">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
